opentreemap/manage_treemap/views/management.py

```diff
@@ -13,15 +14,19 @@
 from django.utils.translation import ugettext as _
```

Comment:
Great subtle find. Here `green_infrastructure_labels` is used as a value for `label_dict`. The other value of `label_dict` uses `ugettext` (`_`) for translation, so I think it's safe to leave as-is. `ugettext` is recommended for views rather than `ugettext_lazy` since different view loads may require different languages.